nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/SFTPTransfer.java

```diff
@@ -642,6 +735,38 @@ public Socket createSocket(InetAddress inetAddress, int i, InetAddress inetAddre
         return sftpClient;
     }
 
+    void restrictSSHOptions(Config config) {
```

Comment:
Recommend renaming to something along the lines of `setAlgorithms` or `setConfig` since this is already in the context of an SSH class.
```suggestion
    private void setConfig(final Config config) {
```